src/main/java/graphql/normalized/ExecutableNormalizedField.java

```diff
@@ -199,13 +291,12 @@ public String getSingleObjectTypeName() {
         return objectTypeNames.iterator().next();
     }
 
+    @Deprecated // using this is ALMOST always wrong
```

Comment:
Deprecating these, we should never use them. As we discussed on Slack last week too.

Reply:
same as the others: lets remove it